src/components/homepage/NewSectionCreator.jsx

```diff
@@ -5,13 +5,57 @@ import Dropdown from "components/Dropdown"
 
 import elementStyles from "styles/isomer-cms/Elements.module.scss"
 
-const NewSectionCreator = ({ createHandler, hasResources }) => {
+// Section constructors
+const defaultResourcesSection = () => ({
+  resources: {
+    title: "Resources Section Title",
+    subtitle: "Resources Section Subtitle",
+    button: "Resources Button Name",
+  },
+})
+
+const defaultInfobarSection = () => ({
+  infobar: {
+    title: "Infobar Title",
+    subtitle: "Infobar Subtitle",
+    description: "Infobar description",
+    button: "Button Text",
+    url: "", // No default value so that no broken link is created
+  },
+})
+
+const defaultInfopicSection = () => ({
+  infopic: {
+    title: "Infopic Title",
+    subtitle: "Infopic Subtitle",
+    description: "Infopic description",
+    button: "Button Text",
+    url: "", // No default value so that no broken link is created
+    image: "", // Always blank since the image modal handles this
+    alt: "Image alt text",
+  },
+})
+
+const enumSection = (type) => {
+  switch (type) {
+    case "resources":
+      return defaultResourcesSection()
+    case "infobar":
+      return defaultInfobarSection()
+    case "infopic":
+      return defaultInfopicSection()
+    default:
+      return defaultInfobarSection()
```

Comment:
I'm wondering if we should leave out a default entirely, so it's easier to catch if we're providing an invalid section type

Reply:
valid point, thanks [b58e761](https://github.com/isomerpages/isomercms-frontend/pull/804/commits/b58e7617fff219d7f3ea19c6708f851f52fbe3db)!